bindings/python/stp/library_path.py.in

```diff
@@ -27,6 +27,19 @@
 # a good idea.
 
 # Search paths for the stp shared library
-PATHS = [
-    '@CMAKE_LIBRARY_OUTPUT_DIRECTORY@/@LIBSTP_FILENAME@'
+
+import os
+import sys
+
+path_var = (
+    "PATH" if sys.platform in ("win32", "cygwin")
+    else "DYLD_LIBRARY_PATH" if sys.platform == "darwin"
+    else "LD_LIBRARY_PATH"
+)
+
+PATHS = ["@CMAKE_LIBRARY_OUTPUT_DIRECTORY@/@LIBSTP_FILENAME@"] + [
+    os.path.join(directory, "@LIBSTP_FILENAME@")
+    for directory in os.environ.get(path_var, "").split(os.pathsep)
 ]
+
+# EOF
```

Comment:
What's this for?

Reply:
The code formatter I use in Python removes the last line of the file, which makes it "hard" to jump the last line in the file (shift+G takes you to the end of the last line and not the last line). It is some "coding standard" I picked-up on.

Reply:
Gone.